### ISSUE TITLE
[Snappi] test case failure in test_multidut_pfcwd_runtime_traffic_with_snappi.py with 400G port and fabric card

### ISSUE THREAD

**Issue by sdszhang** [sonic-net/sonic-mgmt#13916]
### Issue Description

test failed in test_multidut_pfcwd_runtime_traffic_with_snappi.py.
Seems it's because of fabric interface is 200G and test port is 400G. When sending traffic at 400G, half of the packets will be dropped. While test case is expecting no packet drop in this case, hence test failed.

### Results you see

```
rows = <snappi.snappi.FlowMetricIter object at 0x7fa9f9912d40>, speed_gbps = 400
data_flow_dur_sec = 15, data_pkt_size = 1024, tolerance = 0.05

    def __verify_results(rows, speed_gbps, data_flow_dur_sec, data_pkt_size, tolerance):
        """
        Verify if we get expected experiment results

        Args:
            rows (list): per-flow statistics
            speed_gbps (int): link speed in Gbps
            data_flow_dur_sec (int): duration of data flows in second
            data_pkt_size (int): size of data packets in byte
            tolerance (float): maximum allowable deviation

        Returns:
            N/A
        """
        data_flow_rate_percent = int(100 / len(rows))

        for row in rows:
            flow_name = row.name
            tx_frames = row.frames_tx
            rx_frames = row.frames_rx
            logger.info('Flow Name : {} , Tx Frames : {}, Rx Frames : {}'.format(flow_name, tx_frames, rx_frames))

>           pytest_assert(tx_frames == rx_frames, "{} packets of {} are dropped".
                          format(tx_frames - rx_frames, flow_name))
E           Failed: 49260174 packets of Data Flow Prio 1 are dropped

data_flow_dur_sec = 15
data_flow_rate_percent = 14
data_pkt_size = 1024
flow_name  = 'Data Flow Prio 1'
row        = <snappi.snappi.FlowMetric object at 0x7fa9f9912600>
rows       = <snappi.snappi.FlowMetricIter object at 0x7fa9f9912d40>
rx_frames  = 51314538
speed_gbps = 400
tolerance  = 0.05
tx_frames  = 100574712

snappi_tests/multidut/pfcwd/files/pfcwd_multidut_runtime_traffic_helper.py:250: Failed
```

### Results you expected to see

Success

### Is it platform specific

generic

**Comment by sdszhang** [comment]
As discussed in the community meeting, @selldinesh to work on this.

**Comment by sdszhang** [comment]
@rraghav-cisco to work on this.

**Comment by rraghav-cisco** [comment]
ETA for a PR Aug 16th.

**Comment by sdszhang** [comment]
Fixed in #14029
